lib/rules/require-iframe-title.js

```diff
@@ -40,7 +40,7 @@ module.exports = class RequireIframeTitle extends Rule {
           } else if (title.value.type === 'TextNode') {
             let value = title.value.chars.trim();
             let matchIndex = -1;
-            let [, existingNode] = this.knownTitles.filter(([val], index) => {
+            let [, existingNode] = this.knownTitles.find(([val], index) => {
```

Comment:
This change seems unrelated to `.html` files (though it seems fine to do, it should just be a separate PR).

Reply:
it was autofixed by `yarn lint:js --fix`

Reply:
O_o why? Seems like that should have caused linting failures on master then, right?

Reply:
It may be unicorn rules warning fix

Reply:
reverted autofix
![image](https://user-images.githubusercontent.com/1360552/90101998-616ae280-dd48-11ea-93d6-5a836488c882.png)